### ISSUE TITLE
support jsx-no-literals on specific attributes

### ISSUE THREAD

**Issue by davidwkeith** [yannickcr/eslint-plugin-react#2486]
For `react/jsx-no-literals` the default likely should not allow string literals on "title" and "alt" attributes, but allow configuration for component patterns that pass user strings in other ways.

**Comment by ljharb** [comment]
such as?

**Comment by ljharb** [comment]
All HTML element attributes that aren’t enums should always be checked by this rule, I’d think.

**Comment by davidwkeith** [comment]
Makes sense to me. Just noticed a few alt and title attributes that were missed.

**Comment by ljharb** [comment]
Can you provide example code, how you expect it to be linted, and how it is?

**Comment by davidwkeith** [comment]
yep

Fail:
```jsx
<img alt='alt string' {...props} />
<figure title='title string' {...props} />
```

Pass:
```jsx
<img alt={translate(strings.altString)} {...props} />
<figure title={translate(strings.titleString)} {...props} />
```
